[INFO -- 2018-09-25 10:02:23]
Starting flip_value script
version: 1.1
[INFO -- 2018-09-25 10:02:23]
Initialization strings:
[INFO -- 2018-09-25 10:02:23]
(1) file /tmp/knapsack/ga
[INFO -- 2018-09-25 10:02:23]
initSignal:1
[INFO -- 2018-09-25 10:02:23]
endSignal:3.5
[INFO -- 2018-09-25 10:02:23]
faultModel:1
[EXCEPTION -- 2018-09-25 10:02:25]
pythonException: Unable to exit loop in struct fields; Exiting wihtout making a bit flip
[INFO -- 2018-09-25 10:02:25]
Backtrace BEGIN:
#0  __random_r (buf=buf@entry=0x7ffff7832720 <unsafe_state>, result=result@entry=0x7fffffffd804) at random_r.c:380
#1  0x00007ffff7494a22 in __random () at random.c:293
#2  0x00007ffff7494ea9 in rand () at rand.c:27
#3  0x0000555555554eb3 in rand_bit () at knapsackga.cpp:20
#4  0x000055555555514f in crossover (pai=std::vector of length 200, capacity 200 = {...}, mae=std::vector of length 200, capacity 200 = {...}, filho1=std::vector of length 0, capacity 0, filho2=std::vector of length 0, capacity 0) at knapsackga.cpp:74
#5  0x000055555555548d in genetic_algorithm () at knapsackga.cpp:104
#6  0x00005555555558d7 in main () at knapsackga.cpp:137

375	in random_r.c

Backtrace END
Fliping array at pos: 5
Memory content before bitflip:0x4d0x390x4b0xdd
Fault Model: Double bit-flip
base address to flip value: 0x7ffff78321b4 
address max offset: 4
Memory content after  bitflip:0x4d0x990x4b0xdd
frame name: __random
symbol name: randtbl
symbol filename: random.c
symbol line: 146
value: {3, 781116012, 660273869, -162061166, 227818929, -582248115, 203360583, 1295181728, -1871030076, -1956737807, -1171153434, -185921892, 918694676, -1619156898, 1062930419, 1257946405, -299098999, -1065362647, -1243010644, -2137103877, -83715444, 250225202, -113842981, -228497423, 1009322103, 2021751560, -45544622, 2039810360, 2043913220, 116860462, 1155136613, -1953730341}
value address: 0x7ffff78321a0 <randtbl>
Type: The type is an array.
Type sizeof: 128
Field name: None
Field Type: A range type, that is, an integer type with bounds.
Field Type sizeof: 8
Field Type range: (0, 31)

[INFO -- 2018-09-25 10:02:25]
Fault Injection Successful after 2.4514498710632324s
[INFO -- 2018-09-25 10:02:27]
event type: exit
[INFO -- 2018-09-25 10:02:27]
exit code: 0
